### ISSUE TITLE
CentOS and Debian CI failing in kata 2.0

### ISSUE THREAD

**Issue by GabyCT** [kata-containers/tests#2910]
I have seen that the CentOS and Debian CI are failing in kata 2.0 both of them are showing issues in the kubernetes tests (they are using CRIO).
In the case of CentOS http://jenkins.katacontainers.io/view/Kata%202.0/job/kata-containers-2.0-tests-centos-PR/, the `ngnix` test is failing
```
05:02:40 not ok 1 Verify nginx connectivity between pods
05:02:40 # (from function `waitForProcess' in file ../../.ci/lib.sh, line 186,
05:02:40 #  in test file nginx.bats, line 41)
05:02:40 #   `waitForProcess "$wait_time" "$sleep_time" "$cmd"' failed
05:02:40 # Image is up to date for docker.io/library/busybox@sha256:2ca5e69e244d2da7368f7088ea3ad0653c3ce7aaccd0b8823d11b0d5de956002
05:02:40 # Image is up to date for docker.io/library/nginx@sha256:881169baf03885268b54eb07c673bc27f394b263cb728dfd86ff2b65b3450932
05:02:40 # INFO: k8s configured to use runtimeclass
05:02:40 # deployment.apps/nginx-deployment created
05:02:40 # deployment.apps/nginx-deployment condition met
05:02:40 # service/nginx-deployment exposed
05:02:40 # pod/test-nginx created
05:02:40 # deployment.apps "nginx-deployment" deleted
05:02:40 # service "nginx-deployment" deleted
05:02:40 # pod "test-nginx" deleted
05:02:40 Failed at 84: bats "${K8S_TEST_ENTRY}"
05:02:40 [reset] Reading configuration from the cluster...
05:02:40 [reset] FYI: You can look at this config file with 'kubectl -n kube-system get cm kubeadm-config -oyaml'
05:02:40 [preflight] Running pre-flight checks
05:02:40 [reset] Removing info for node "centos8-51c4c0" from the ConfigMap "kubeadm-config" in the "kube-system" Namespace
05:02:40 W1001 10:02:40.350968  129578 removeetcdmember.go:61] [reset] failed to remove etcd member: error syncing endpoints with etc: etcdclient: no available endpoints
05:02:40 .Please manually remove this etcd member using etcdctl
05:02:40 [reset] Stopping the kubelet service
05:02:40 [reset] Unmounting mounted directories in "/var/lib/kubelet"
```

Now in the case of Debian http://jenkins.katacontainers.io/view/Kata%202.0/job/kata-containers-2.0-tests-debian-PR/,
we have issue with `Verify nginx connectivity between pods` test
```
04:44:21 not ok 1 Verify nginx connectivity between pods
04:44:21 # (from function `waitForProcess' in file ../../.ci/lib.sh, line 186,
04:44:21 #  in test file nginx.bats, line 41)
04:44:21 #   `waitForProcess "$wait_time" "$sleep_time" "$cmd"' failed
04:44:21 # Image is up to date for docker.io/library/busybox@sha256:2ca5e69e244d2da7368f7088ea3ad0653c3ce7aaccd0b8823d11b0d5de956002
04:44:21 # Image is up to date for docker.io/library/nginx@sha256:881169baf03885268b54eb07c673bc27f394b263cb728dfd86ff2b65b3450932
04:44:21 # INFO: k8s configured to use runtimeclass
04:44:21 # deployment.apps/nginx-deployment created
04:44:21 # deployment.apps/nginx-deployment condition met
04:44:21 # service/nginx-deployment exposed
04:44:21 # pod/test-nginx created
04:44:21 # deployment.apps "nginx-deployment" deleted
04:44:21 # service "nginx-deployment" deleted
04:44:21 # pod "test-nginx" deleted
04:44:21 Failed at 84: bats "${K8S_TEST_ENTRY}"
04:44:21 [reset] Reading configuration from the cluster...
04:44:21 [reset] FYI: You can look at this config file with 'kubectl -n kube-system get cm kubeadm-config -oyaml'
04:44:21 [preflight] Running pre-flight checks
04:44:21 [reset] Removing info for node "debian10-azuref01b60" from the ConfigMap "kubeadm-config" in the "kube-system" Namespace
04:44:21 W1001 09:44:21.855730    5533 removeetcdmember.go:61] [reset] failed to remove etcd member: error syncing endpoints with etc: etcdclient: no available endpoints
04:44:21 .Please manually remove this etcd member using etcdctl
04:44:21 [reset] Stopping the kubelet service
04:44:21 [reset] Unmounting mounted directories in "/var/lib/kubelet"
```

**Comment by jodh-intel** [comment]
That's an odd combination of distros to be failing.

**Comment by GabyCT** [comment]
no PR has been identified yet (introduce this issue)

**Comment by GabyCT** [comment]
Closing this issue as it has been fixed in kata 2.0
